test/core/Rectangle.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-describe('PIXI.Rectangle', function ()
+describe('PIXI.Rectangle', () =>
 {
-    it('should create a new rectangle', function ()
+    it('should create a new rectangle', () =>
```

Comment:
@bigtimebuddy I was gonna note that, but I couldn't remember why we do it.  Is it worth having that demonstration of the mocha scope vs fat arrows that you posted at some stage in the wiki?